### DISCUSSION TITLE
Welcome to OpenModSim Discussions!

### DISCUSSION THREAD

**Discussion by sanny32** [sanny32/OpenModSim#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RaimundSch** [comment]
Hello, thank you for making your program available to us.

I have a question:

I'm using Home Assistant to write values ​​to the holding register.

That works, but the changes to the values ​​aren't displayed in the view.

The changes are only displayed when I switch to, for example, 01: Coil Status and then back to 03: Holding Register.

Is this intended, or am I doing something wrong?

Best regards,

Raimund
